src/grt/src/fastroute/src/maze.cpp

```diff
@@ -55,16 +55,16 @@ static int right_index(int i)
   return 2 * i + 2;
 }
 
-void FastRouteCore::fixEmbeddedTrees()
-{
-  // check embedded trees only when maze router is called
-  // i.e., when running overflow iterations
-  if (overflow_iterations_ > 0) {
-    for (const int& netID : net_ids_) {
-      checkAndFixEmbeddedTree(netID);
-    }
-  }
-}
+// void FastRouteCore::fixEmbeddedTrees()
+// {
+//   // check embedded trees only when maze router is called
+//   // i.e., when running overflow iterations
+//   if (overflow_iterations_ > 0) {
+//     for (const int& netID : net_ids_) {
+//       checkAndFixEmbeddedTree(netID);
+//     }
+//   }
+// }
```

Comment:
You can remove this commented code too. This function will not be used again in the future.